getting-started-credentials.md

```diff
@@ -2,7 +2,11 @@
 
 copyright:
   years: 2015, 2019
-lastupdated: "2018-12-12"
+lastupdated: "2019-03-08"
+
+keywords: Cloud Foundry, authentication,service credentials
```

Comment:
I don't whether this matters at all, but you have a space after the comma that precedes the text of the second keyword string.  You haven't had such spaces elsewhere.